src/Model/Item.php

```diff
@@ -383,16 +383,10 @@ public static function selectToArray(array $fields = [], array $condition = [],
 		$result = self::select($fields, $condition, $params);
 
 		if (is_bool($result)) {
-			return $result;
-		}
-
-		$data = [];
-		while ($row = self::fetch($result)) {
-			$data[] = $row;
+			return [];
 		}
-		DBA::close($result);
 
-                return $data;
+        return DBA::toArray($result);
```

Comment:
You mustn't use ```DBA::toArray(``` here. We have to rely upon ```Item::fetch(``` here, since that function is doing some magic here.

Reply:
Ha, I didn't notice the `$this->fetch()`.

Reply:
That's why I would like to have it into separate PRs :-)

Reply:
Reverted the faulty change.